assets/scripts/main.js

```diff
@@ -50,15 +60,31 @@ function setScaleForPhone () {
   headEls[0].appendChild(meta)
 }
 
-setScaleForPhone()
+if (unsupportedBrowser()) {
+  let loadingContainer = document.getElementById('loading')
+  loadingContainer.style.display = 'none'
+  let body = document.getElementsByTagName('body')[0]
+  body.style.background = '#a9ccdb'
+  let div = document.createElement('div')
+  div.style.cssText = `color: #000; justify-content: center; 
+                       display: block; margin: 150px 20px; text-align: center;
+                      `
+  div.innerHTML = `
+    <h1>Oops! Your version of Internet Explorer is not supported</h1>
+    <p>Download version 9 and above. <a href='https://www.microsoft.com/en-us/download/internet-explorer.aspx' target="_blank" >Click here to download</a>
+  `
+  body.appendChild(div)
```

Comment:
We'll also need a different approach for rendering an unsupported browser message. Browser support for various JavaScript and DOM methods can be all over the place. We might also want to display such a message if the user has JavaScript turned off. In this case the most disaster-proof method is to render the unsupported message directly in HTML by default, and only remove it after React mounts.